src/civic/index.js

```diff
@@ -128,6 +128,8 @@ const translateRelevance = (evidenceType, evidenceDirection, clinicalSignificanc
                 case 'Sensitivity/Response': {
                     return 'no response';
                 }
+
+                case 'Resistance': { return 'no resistance'; }
```

Comment:
super minor so can skip if its a hassle, but can you split the bracket to match the formatting of the other cases? 

```
case 'Resistance': { 
    return 'no resistance'; 
}
```